langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -357,6 +509,19 @@ public isolated function forEach(xml<ItemType> x, @isolatedParam function(ItemTy
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml codingBooks = xml `<scripting>JS</scripting><programming>Ballerina</programming>
+#                   <programming>Python</programming><scripting>Perl</scripting>`;
+# 
+# xml programmingBooks = codingBooks.filter(function (xml xmlItem) returns boolean {
```

Comment:
```suggestion
# xml programmingBooks = codingBooks.filter(isolated function (xml xmlItem) returns boolean {
```

Reply:
Similar to: https://github.com/ballerina-platform/ballerina-lang/pull/38981#discussion_r1053241010